amgut/lib/util.py

```diff
@@ -169,8 +170,9 @@ def survey_vioscreen(survey_id):
     """Return a formatted text block and URL for the external survey"""
     tl = text_locale['human_survey_completed.html']
     embedded_text = tl['SURVEY_VIOSCREEN']
+    user_info = ag_data.get_person_info(survey_id)
     url = ("https://vioscreen.com/remotelogin.aspx?Key=%s&RegCode=KLUCB" %
-           url_escape(encrypt_key(survey_id)))
+           url_escape(encrypt_key(survey_id, user_info)))
```

Comment:
@beaugunderson, can you comment as to why this was done? @squirrelo, pinging you to keep you in the loop since you're most familiar with this interaction.